src/main/webapp/app/exercises/shared/feedback-suggestion/exercise-feedback-suggestion-options.component.html

```diff
@@ -0,0 +1,38 @@
+@if ((isAthenaEnabled$ | async) && modulesAvailable) {
+    <div class="form-group">
+        <div class="form-check">
+            <input
+                type="checkbox"
+                class="form-check-input"
+                name="feedbackSuggestionsEnabledCheck"
+                [checked]="!!exercise.feedbackSuggestionModule"
+                [disabled]="inputControlsDisabled()"
+                id="feedbackSuggestionsEnabledCheck"
+                (change)="toggleFeedbackSuggestions($event)"
+            />
+            <label
+                [ngStyle]="getCheckboxLabelStyle()"
+                class="form-control-label"
+                for="feedbackSuggestionsEnabledCheck"
+                jhiTranslate="artemisApp.exercise.feedbackSuggestionsEnabled"
+            ></label>
```

Comment:
```suggestion
            />
```

Reply:
Intellij complains that the empty tag doesn't work in some browsers

Reply:
You're right. That's a discussion I had recently. "All browsers except Netscape 4" 😂